azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -132,7 +141,19 @@ internal open class CallCompositeActivity : AppCompatActivity() {
         }
         updatableOptionsManager.start()
         setContentView(R.layout.azure_communication_ui_calling_activity_call_composite)
+        if (Build.VERSION.SDK_INT >= 35) {
```

Comment:
please encapsulate API version  specific logic into it's own function when possible.